WebApp/WebContent/data_file/upload.xhtml

```diff
@@ -10,10 +10,10 @@
 	<ui:define name="localHead">
 		<h:outputScript name="script/dataFiles.js"/>
 		<h:outputStylesheet name="style/dataFiles.css"/>
+		<h:outputStylesheet name="grid/grid.css" library="primefaces" />
 	</ui:define>
-	
```

Comment:
Note that the expression for getting the instrument name has changed. I think you need to rebase from upstream/alpha2.02 to get the updated code.